vscode/README.md

```diff
@@ -389,39 +389,47 @@ Please note that only Docker is officially supported as a backend by the Dev Con
 
 ## Telemetry
 
-On its own, the Ruby LSP does not collect any telemetry by default, but it does support hooking up to a private metrics
-service if desired.
+The Ruby LSP does not collect any telemetry by default, but it supports hooking up to a private metrics service if
+desired. This can be useful if you'd like to understand adoption, performance or catch errors of the Ruby LSP within
+your team or company.
 
-In order to receive metrics requests, a private plugin must export the `ruby-lsp.getPrivateTelemetryApi` command, which
-should return an object that implements the `TelemetryApi` interface defined
-[here](https://github.com/Shopify/ruby-lsp/blob/main/vscode/src/telemetry.ts).
+To collect metrics, another VS Code extension (typically a private one) should define the command
+`getTelemetrySenderObject`. This command should return an object that implements the
```

Comment:
(nit) we control this name, right? Could it just be `getTelemetrySender`?

Reply:
That would be a breaking change in our own private extension which already uses `getTelemetrySenderObject`. Unless you feel really strongly about this, I'd prefer to avoid changing it.

Reply:
Sure.